Automattic/newspack-ads
# update dependencies to fix build issues

We'd use a larger updates such as https://github.com/Automattic/newspack-newsletters/pull/625, but this is just to make CI jobs pass and unblock the other PRs.

Closes #232

## How to test

1. Make sure `npm build` & `npm start` work 
2. Do a basic smoke test, nothing should change in functionality
